stage_tester/dev.conf

```diff
@@ -0,0 +1,8 @@
+[load_test]
+
+offline_token=..
+login_url=https://sso.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token
```

Comment:
If it is stage test, why is it using production SSO?

Reply:
that is just an example url, moreover I think the folder name  is unfortunate, or I put the locust file in the wrong place, in fact we can test several environments by setting those vars...

However if stage has a different log in mechanism we must implement it (this has been tested only against prod